packages/replay-next/src/contexts/FocusContext.tsx

```diff
@@ -22,11 +26,16 @@ import {
   imperativelyGetClosestPointForTime,
   preCacheExecutionPointForTime,
 } from "../suspense/ExecutionPointsCache";
-import { TimeRange } from "../types";
+import { TimeRange as TimeRangeArray } from "../types";
 import { SessionContext } from "./SessionContext";
 
 const FOCUS_DEBOUNCE_DURATION = 250;
 
+interface TimeRange {
+  begin: number;
+  end: number;
+}
```

Comment:
We already have a type defined for this in a more shared location. Can we use it instead?

https://github.com/replayio/devtools/blob/95c9148aa9a1676a0bbee2fc0327167fcad43ef4/packages/replay-next/src/types.ts#L4

We also have some redundant ones that would be nice to just delete:

https://github.com/replayio/devtools/blob/95c9148aa9a1676a0bbee2fc0327167fcad43ef4/src/ui/utils/app.ts#L2-L5

https://github.com/replayio/devtools/blob/95c9148aa9a1676a0bbee2fc0327167fcad43ef4/src/ui/state/timeline.ts#L3-L6

Reply:
> I prefer the object type over the array type

Yeah that makes sense. I would like to just get rid of the array version, if we could.

> And I don't want to import a type from src/ui into replay-next.

Yeah totally, we shouldn't do that.